build-system/tasks/visual-diff.js

```diff
@@ -428,7 +428,7 @@ async function snapshotWebpages(percy, page, webpages, config) {
 
     await enableExperiments(page, webpage['experiments']);
 
-    const snapshotOptions = SNAPSHOT_OPTIONS;
+    const snapshotOptions = Object.assign({}, SNAPSHOT_OPTIONS);
```

Comment:
For readability, maybe rename constants like `SNAPSHOT_OPTIONS` to `DEFAULT_SNAPSHOT_OPTIONS`?

Reply:
Also, yeah, JS consts aren't immutable. They just can't be reassigned. Ugly bug :(

Reply:
Done